From ce939be3d119ba9162302d6598b3dc6b6c28a62c Mon Sep 17 00:00:00 2001
From: Jeremy Linton <lintonrjeremy@gmail.com>
Date: Wed, 26 Apr 2017 21:59:29 -0500
Subject: [PATCH] arm64: mmc drivers

It is common for arm64 devices to boot from mmc & sd devices
we need to assure that those drivers make it to the initrd.

Signed-off-by: Jeremy Linton <lintonrjeremy@gmail.com>
---
 modules.d/90kernel-modules/module-setup.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/90kernel-modules/module-setup.sh b/modules.d/90kernel-modules/module-setup.sh
index ee6ba5d4..2ed9cfca 100755
--- a/modules.d/90kernel-modules/module-setup.sh
+++ b/modules.d/90kernel-modules/module-setup.sh
@@ -26,10 +26,12 @@ installkernel() {
 
 	if [[ "$(uname -m)" == arm* || "$(uname -m)" == aarch64 ]]; then
             # arm/aarch64 specific modules
+            _blockfuncs+='|dw_mc_probe|dw_mci_pltfm_register' 
             instmods \
                 "=drivers/clk" \
                 "=drivers/dma" \
                 "=drivers/i2c/busses" \
+                "=drivers/mfd" \
                 "=drivers/phy" \
                 "=drivers/power" \
                 "=drivers/regulator" \
